sdk/src/main/java/io/dapr/client/DaprHttp.java

```diff
@@ -317,7 +318,12 @@ private static DaprError parseDaprError(byte[] json) throws IOException {
     if ((json == null) || (json.length == 0)) {
       return null;
     }
-    return OBJECT_MAPPER.readValue(json, DaprError.class);
+
+    try {
+      return OBJECT_MAPPER.readValue(json, DaprError.class);
+    } catch (JsonParseException e) {
+      throw new DaprException("UNKNOWN", new String(json, StandardCharsets.UTF_8));
+    }
```

Comment:
Is this supposed to be here? What are the cases where this is called?

Reply:
This can happen if Dapr does not unescape correctly and returns a bad response at the HTTP server prior to the request being processed by dapr logic. I reproduced this with daprd with a bad patch from another PR - so I thought it would be good to have a better experience in the SDK if that kind of corner case happens.